Automattic/help-ukraine-widget
# Internationalization: improve i18n support

#### Changes proposed in this Pull Request:

﻿- i18n: add missing translation function around explanation
- Update escaping functions in frontend output.
- Ensure settings can be translated
- Bump version and add elements to plugin readme header.

#### Testing instructions:

- Check that the settings are still properly displayed in wp-admin.
- Check that the script enqueued on the page is still enqueued properly.
